book/http_cache.rst

```diff
@@ -1031,21 +1064,36 @@ matter), Symfony uses the standard ``render`` helper to configure ESI tags:
 
     .. code-block:: jinja
 
+        {# app/Resources/views/about.html.twig #}
```

Comment:
`DefaultController` is basically due to a lack of imagination. And so, I think we can try a little harder in many areas and give things more meaningful names.